src/main/java/dev/joopie/jambot/repository/album/AlbumFinder.java

```diff
@@ -0,0 +1,25 @@
+package dev.joopie.jambot.repository.album;
+
+import dev.joopie.jambot.models.album.Album;
+import dev.joopie.jambot.repository.base.BaseFinder;
+import io.ebean.DB;
+
+import java.util.List;
+
+public class AlbumFinder extends BaseFinder<Album> {
```

Comment:
So I _think_ this is just a matter of taste. My point of view; repositories are responsible for repository task. Updating, saving, deletion. Finder are for fetching data. Also my current way of working now in my day to day job. 

I agree that for little classes like these we _could_ integrate them. But you can imagine if we want some advanced queries in the future, the repository class gets bloated. By having clear separations now, I hope that it will prevent big repository classes in our application. And let's be honest. You only need to call the find() function to get started.